### ISSUE TITLE
Add exposed ports to initial mds.sh for new services

### ISSUE THREAD

**Issue by reavessm** [reavessm/MDS#13]
Whenever we create a new service, we still have to open up a web browser to find what ports to expose.  We can prepopulate this with image defaults by using `docker inspect $conImage`  This may also allow us to search other things like volume mounts.  We can `--format` the previous command to make it play nice with `awk`.

After we prepopulate the image with defaults, the user can change the ports based on the output of `make checkPorts`

**Comment by reavessm** [comment]
You can't inspect remote images, so we need to pull before we inspect.

```bash
# Download image info
docker pull $conImg

# Extract port numbers in space delimited format
docker image inspect -f '{{.Config.ExposedPorts}}' $conImg | sed 's/[^[:digit:][:space:]]//g' 
```

We should probably create a `portArray` and += the above command, similar to `args` in the container specific `mds.sh`.  Then map the ports to themselves.  Eventuallly do the same for volumes, etc.

# Example:

```bash
$ conImg="linuxserver/letsencrypt"
$ docker pull $conImg && docker image inspect -f '{{.Config.ExposedPorts}}' $conImg | sed 's/[^[:digit:][:space:]]//g' 
443 80
```

results in

```bash
args="-d"
args+=" -p 443:443"
args+=" -p 80:80"
```

**Comment by reavessm** [comment]
Fixed with 84a11b03664e0f89df59ecb0fe56f0b423aba3a1

**Comment by reavessm** [comment]
Fixed  84a11b03664e0f89df59ecb0fe56f0b423aba3a1
